owasp_zap_historic_parser/owasp_zap_historical.py

```diff
@@ -45,6 +44,14 @@ def html_parser(filename):
     # Get parser ready
     parser = etree.HTMLParser()
     tree = etree.parse(filename, parser)
+    # See if report has alert table and adjust which locators to use
+    table_count = int(tree.xpath("count(" + TABLE_LOCATOR + ")"))
+    print(table_count)
```

Comment:
Just double checking that you still want this print, not sure if its for logging or debugging